Assets/Huawei/Scripts/ML/LangDetector/HMSMLLanguageDetectionManager.cs

```diff
@@ -0,0 +1,180 @@
+using HmsPlugin;
+using HuaweiMobileServices.ML.Common;
+using HuaweiMobileServices.ML.LanguageDetection;
+using HuaweiMobileServices.ML.LanguageDetection.Cloud;
+using HuaweiMobileServices.ML.LanguageDetection.Local;
+using HuaweiMobileServices.Utils;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+namespace Assets.Huawei.Scripts.ML.LangDetector
+{
+    public enum DetectionMode
```

Comment:
We can relocate this enum to HMSMLLanguageDetectionManager. It is accessible everywhere like this.